manual/guides/guides.md

```diff
@@ -58,51 +58,51 @@ yours) easier.
 
 # Communication
 
-We communicate through many channels. Use the 
-[communication](/manual/guides/communication) guide to understand and be 
-prepared to make use of the various meetings, listhosts, chat media, calendars, 
+We communicate through many channels. Use the
+[communication](/manual/guides/communication) guide to understand and be
+prepared to make use of the various meetings, listhosts, chat media, calendars,
 twitter accounts, and whatnot that tie us together.
 
 # Writing
 
-- [how to write](/manual/guides/writing/)
+- [How to Write](/manual/guides/writing/)
 
```

Comment:
Maybe the separate writing guides deeper in the filestructure could be listed here as well as the overview writing link. That is, the guides for writing a conference abstract, journal article, etc. (at http://arfc.github.io/manual/guides/writing/conf-abs/ etc.) I know it's weird because they're at a deeper level... it just seems like the writing section is lonely here.

Reply:
Does the "Writing" tab look like what you were thinking? I basically just copied the different types of entries and threw them in